councilmatic_core/management/commands/refresh_pic.py

```diff
@@ -31,17 +34,64 @@ def handle(self, *args, **options):
         from boto.s3.key import Key
         from boto.exception import S3ResponseError
 
-        document_urls = self._get_urls()
-        aws_keys = self._create_keys(document_urls)
-
         s3_conn = S3Connection(settings.AWS_KEY, settings.AWS_SECRET)
 
+        document_urls = self._get_urls()
+        aws_keys = self._create_keys(document_urls)
         bucket = s3_conn.get_bucket('councilmatic-document-cache')
-
         bucket.delete_keys(aws_keys)
 
         logger.info(("Removed {} document(s) from the councilmatic-document-cache").format(len(aws_keys)))
 
+    @property
+    def local_now(self):
+        return app_timezone.localize(datetime.datetime.now())
+
+    @property
+    def bills_on_upcoming_agendas(self):
```

Comment:
Using properties for `local_now` and `bills_on_upcoming_agendas` feels not right... Perhaps I should set these attributes in an init method, instead?

Reply:
I like that! One advantage of setting this attribute in `__init__()` is that we won't have to rerun the `EventRelatedEntity` query every time the attribute is accessed. I would just flag that we'll have to [make sure to call the parent `__init__()` method first](https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/#s-methods) due to the particularities of the `BaseCommand` class.